reference/enterprise-api-ref/build-api.markdown

```diff
@@ -0,0 +1,286 @@
+---
+layout: default
+title: Build API
+published: true
+tags: [reference, enterprise, API, build, modules]
+---
+
+The Build API enables you to configure modules easily to your projects.
+
+# Projects API
```

Comment:
The h1 here throws me, I don't think we typically use h1, and this is the only h1 I see on the page. Are there more build related APIs to go in this page that are not the Projects api?

Reply:
I planned to add more APIs related to build here